<?xml version="1.0"?>
<!--
Demo launch file to run single softkinetic camera in stand alone (no other /tf) setup with Rviz
The arguments given are the device indices of the cameras determined by the DepthSense deamon.
-->
<launch>
  <node name="aricc_3d_vision" pkg="aricc_3d_vision" type="aricc_3d_vision_bringup_node" args="0" output="screen" >
    <param name="camera_link"           type="string"    value="base_rgbd_camera_link" />
    <param name="cloud_sub_topic"       type="string"    value="/softkinetic_camera/depth_registered/points" />
    <param name="cloud_pub_topic"       type="string"    value="depth_registered/processed_points" />
  </node>
</launch>
